<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQCompartment.ui,v $ </comment>
<comment>   $Revision: 1.15 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2010/07/16 19:05:16 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
 <comment> All rights reserved. </comment>
 <class>CQCompartment</class>
 <widget class="CopasiWidget" name="CQCompartment" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>448</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="caption" stdset="0" >
   <string>Compartment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="mpLblName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Compartment Name</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QLineEdit" name="mpEditName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="mpLblDim" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Dimensionality</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="mpComboBoxDim" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text" >
         <string>0 - dimensionless</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1D - Length</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>2D - Area</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>3D - Volume</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="mpLblType" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Simulation Type</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="mpComboBoxType" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="mpLblExpression" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="CQExpressionMmlStackedWidget" name="mpExpressionEMW" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QWidget" name="page" />
       <widget class="QWidget" name="page_2" />
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="mpLblInitialValue" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Initial Volume</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="mpEditInitialVolume" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <widget class="QCheckBox" name="mpBoxUseInitialExpression" >
       <property name="text" >
        <string>Use Initial Expression</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="mpLblInitialExpression" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Initial Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2" >
      <widget class="CQExpressionMmlStackedWidget" name="mpInitialExpressionEMW" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_3" />
       <widget class="QWidget" name="page_4" />
      </widget>
     </item>
     <item row="6" column="0" colspan="3" >
      <widget class="Line" name="mpLine1" >
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="mpLblVolume" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Volume</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="QLineEdit" name="mpEditCurrentVolume" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" >
      <widget class="QLabel" name="mpLblRate" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Rate</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1" >
      <widget class="QLineEdit" name="mpEditRate" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" >
      <widget class="QLabel" name="mpLblMetabolites" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Contained Metabolites</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2" >
      <widget class="Q3ListView" name="mpMetaboliteTable" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text" >
         <string>Name</string>
        </property>
        <property name="clickable" >
         <bool>true</bool>
        </property>
        <property name="resizable" >
         <bool>true</bool>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="mpVerticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="mpLine2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="mpBtnCommit" >
       <property name="text" >
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnRevert" >
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnNew" >
       <property name="text" >
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQExpressionMmlStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>CQExpressionMmlStackedWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpEditName</tabstop>
  <tabstop>mpComboBoxDim</tabstop>
  <tabstop>mpComboBoxType</tabstop>
  <tabstop>mpExpressionEMW</tabstop>
  <tabstop>mpEditInitialVolume</tabstop>
  <tabstop>mpBoxUseInitialExpression</tabstop>
  <tabstop>mpInitialExpressionEMW</tabstop>
  <tabstop>mpEditCurrentVolume</tabstop>
  <tabstop>mpEditRate</tabstop>
  <tabstop>mpMetaboliteTable</tabstop>
  <tabstop>mpBtnCommit</tabstop>
  <tabstop>mpBtnRevert</tabstop>
  <tabstop>mpBtnNew</tabstop>
  <tabstop>mpBtnDelete</tabstop>
 </tabstops>
 <includes>
  <include location="global" >string</include>
  <include location="local" >copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCommit</sender>
   <signal>clicked()</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotBtnCommit()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>clicked()</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotBtnDelete()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnNew</sender>
   <signal>clicked()</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotBtnNew()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRevert</sender>
   <signal>clicked()</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotBtnRevert()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpComboBoxType</sender>
   <signal>activated(int)</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpMetaboliteTable</sender>
   <signal>doubleClicked(Q3ListViewItem*)</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotMetaboliteTableCurrentChanged(Q3ListViewItem*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditName</sender>
   <signal>lostFocus()</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotNameLostFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBoxUseInitialExpression</sender>
   <signal>toggled(bool)</signal>
   <receiver>CQCompartment</receiver>
   <slot>slotInitialTypeChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
